### DISCUSSION TITLE
Lyrics support ?

### DISCUSSION THREAD

**Discussion by jairaume** [ciderapp/Apple-Music-Electron#144] [Ideas]
Hello !
As a recent Apple music subscriber, i was previously using spotify and [spicetify](https://github.com/khanhas/spicetify-cli) (spotify customization tool), which has "custom Apps" and "extensions" to add features to the spotify client. 
Could it be possible to have, for example, a "Lyrics" button somewhere in the app linked to [Genius](https://docs.genius.com/) or _**even better**_ [Musixmatch ](https://developer.musixmatch.com/)for synced lyrics ? 
I customized my spicetify to make it as i wanted so i think i will _attempt_ to look and fiddle around the source code, but i think it could be a great feature to add to the app ! 
Anyways, congrats to the dev, i really appreciate the application :)

**Comment by jairaume** [comment]
I also "made" in spicetify a dynamic theme which changed the accent color of all buttons and icons according to the vibrant color of the album cover. The main background color was like a dark grey, tinted to the second vibrant color. That could look slick in the apple music interface too...

**Reply by quacksire** [reply]
Please create an issue and/or a PR with this. It helps us keep track of feature development

**Reply by quacksire** [reply]
#145

**Reply by jairaume** [reply]
all right ! **if** i manage to come up with something working somehow, i'll be glad to share :)
